core/arch/arm/dts/stm32mp13-pinctrl.dtsi

```diff
@@ -16,6 +16,15 @@
 		};
 	};
 
+	rcc_mco_pins_a: rcc-pins-0 {
```

Comment:
see other reamrks => please drop MCO part

Reply:
I'll leave the pinctrl part which is fine for that board. I'll only discard the MCO driver part and its node/compatible in the DT.